docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -549,6 +556,38 @@ v: 1.0, stage: prod   v: 1.1, stage: canary            v: 1.1, stage: canary
 v: 1.0                (none)                           v: 1.0
 ====================  ===============================  ====================
 
+Scaled Locality Weights
+***********************
+
+When both locality weight and locality weight scaling is enabled, the weight
+of each locality will depend on the fraction of endpoints removed from the locality
+due to the subsetting.
+
+Consider the following
```

Comment:
The RST is now quite right here, see how it renders at https://125902-65214191-gh.circle-artifacts.com/0/source/generated/docs/intro/arch_overview/load_balancing.html#load-balancer-subsets